### ISSUE TITLE
Operator OR between queries seems not working properly

### ISSUE THREAD

**Issue by vferretti** [kids-first/kf-portal-ui#1548]
Select Age_at_diagnosis between 1 and 5 years -> the histogram Age at diagnosis shows one bar which is okay

Start a new query and select age_at_diagnosis between 5 and 10 -> histogram is good too

Do query 1 OR query 2, we get an empty histogram (we should have two columns)

The most frequent diagnosis chart is also empty which is not rigth

## Screenshot
As you can see, two of the charts are empty, but should be filled with some data.
![screenshot_2019-04-15_15-51-50_combine_or_age_at_diagnosis](https://user-images.githubusercontent.com/293677/56160987-7b4eb680-5f96-11e9-89f8-784d17c5ef64.png)

**Comment by aalex** [comment]
@jberube @nyanofthemoon Let us know if this issue depends on a tech debt one.

**Comment by aalex** [comment]
@hlminh2000 Are we blocked by limitations in the backend for this one? (SQON, Arranger, or another element)

**Comment by nyanofthemoon** [comment]
@aalex Currently not supporting "multiple" operations within the same query- we can address this issue once we refactor the SQON Builder~

**Comment by rosibaj** [comment]
@aalex @nyanofthemoon Vincent asked that @hlminh2000 do this one so i added it to the Kanban for OICR and assigned Minh. 

@aalex please let me know if you need me to do something else.

**Comment by hlminh2000** [comment]
This looks like a back-end sqon resolution issue, of how the sqon is translated to an elasticsearch query

**Comment by aalex** [comment]
Yes. Also, if there are mixed operators (AND and OR) in the same query, AND should have precedence over OR.

**Comment by rosibaj** [comment]
@aalex Is that additional point mentioned in another ticket somewhere?

**Comment by aalex** [comment]
Mixing AND and OR operators are for another ticket: you are right. My mistake.
